spring-cloud-stream-core-docs/src/main/asciidoc/spring-cloud-stream-overview.adoc

```diff
@@ -678,6 +655,202 @@ boolean result = pollableSource.poll(received -> {
 		}, new ParameterizedTypeReference<Map<String, Foo>>() {});
 ----
 
+[[spring-cloud-stream-overview-error-handling]]
+==== Error Handling
+
+Errors happen and Spring Cloud Stream provides several flexible mechanisms to handle them.
+The error handling comes in two flavors
+
+	- *system* - the error handling is delegated to the binder (re-queue, DLQ etc). Note that the techniques are dependent on binder implementation and
+	capability of the underlying messaging middleware.
+
+	- *application* - the error handling is done within the application (custom error handler)
+
+Spring Cloud Stream uses https://github.com/spring-projects/spring-retry[Spring Retry] library to facilitate successful message processing. See <<Retry Template>> for more details.
+However, when all fails the exceptions thrown by the message handlers are propagated back to the binder.  At that point binder invokes custom error handler or communicates
+the error back to the messaging system (re-queue, DLQ etc).
+
+===== Application Error Handling
+
+Let’s first look at the application level error handling.
+
+For each input binding, Spring Cloud Stream creates a dedicated error channel with the following semantics `<channel-name>.<group-name>.errors`.
+If such channel has a subscriber all errors will be sent to that subscriber.
```

Comment:
Replace with "If such a channel has a subscriber, all errors are sent to that subscriber."